assets/sass/patterns/atoms/buttons.scss

```diff
@@ -85,5 +85,55 @@
 .button--outline-inactive {
   @include colors($color-text--reverse, #909090, #909090);
   border-width: 2px;
+
   cursor: not-allowed;
 }
+
+.button--speech-bubble {
+  display: none;
+}
+
+// TODO: could build with js so no need for js-only show defined in css
+.js {
+
+  .button--speech-bubble {
+    @include colors($color-primary);
+    display: block;
+    font-family: $font-primary;
+    @include font-size-and-vertical-height(48, 36);
+    @include height(36);
+    @include padding(12 0 0);
+    position: relative;
+    @include width(42);
+
+    &:after {
```

Comment:
Is it possible to make this clickable too?

Reply:
I've played with some hacky approaches and the answer is yes in theory, but want to find a nicer approach to it.